Marajoara.Cinema.Management/Marajoara.Cinema.Management.FrontEnd/src/app/components/user/user-account-form/user-account-form.component.scss

```diff
@@ -0,0 +1,18 @@
+.form-group-flex-row mat-checkbox {
+  margin-bottom: 15px;
```

Comment:
No need for this class, there is no checkbox on this form

Reply:
Resolved